### ISSUE TITLE
is it a good idea to make a dock container without such setup hassel?

### ISSUE THREAD

**Issue by raptoravis** [nickjj/dotfiles#5]
all these setup of vim/tmux/fzf etc. are really handy and charming. but the setup process is still a bit tedious though it is already quite smooth.

is it a good idea to make a dock container without such setup hassel?

**Comment by nickjj** [comment]
Hi,

Did you have trouble following the README or run into any errors?

It should be just copy / pasting a couple of commands. In theory if you combined them into 1 command you could make it a single command. I didn't do that combination step because I didn't want to enforce every single tool I use on everyone.

I don't like the idea of putting everything into a Docker container because it's not just running a single process. Plus there would need to be many different volumes mounts to retain things like Vim's undo history, tmux sessions. Then there's things like having to volume mount your entire file system to use fzf or figuring out how to persist a tmux session if your container stops.

Basically Docker is great for a lot of things but I don't think it's a good match for trying to roll up your whole development environment in. But I am open to improving the installation process without Docker.
